src/ioctls/system.rs

```diff
@@ -267,8 +272,8 @@ impl Kvm {
     /// ```
     ///
     #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-    pub fn get_emulated_cpuid(&self, max_entries_count: usize) -> Result<CpuId> {
-        self.get_cpuid(KVM_GET_EMULATED_CPUID(), max_entries_count)
+    pub fn get_emulated_cpuid(&self, num_entries: usize) -> Result<CpuId> {
```

Comment:
I think we should also update the documentation of the function to specify that it returns an error when `num_entries` > KVM_MAX_CPUID_ENTRIES. Same for `get_supported_cpuid`.